bitnami/sealed-secrets/templates/deployment.yaml

```
@@ -85,7 +89,54 @@ spec:
           {{- else }}
           args:
             - "--key-prefix"
-            - {{ printf "%s-key" (include "common.names.fullname" .) | quote }}
+            - {{ printf "%s" (include "sealed-secrets.secretName" .) | quote }}
+            {{- if .Values.updateStatus }}
+            - "--update-status"
+            {{- end }}
+            {{- if .Values.skipRecreate }}
+            - "--skip-recreate"
+            {{- end }}
+            {{- if .Values.keyrenewperiod }}
+            - "--key-renew-period"
+            - {{ .Values.keyrenewperiod | quote }}
+            {{- end }}
+            {{- if .Values.rateLimit }}
+            - "--rate-limit"
+            - {{ .Values.rateLimit | quote }}
+            {{- end }}
+            {{- if .Values.rateLimitBurst }}
+            - "--rate-limit-burst"
+            - {{ .Values.rateLimitBurst | quote }}
+            {{- end }}
+            {{- if .Values.additionalNamespaces }}
+            - "--additional-namespaces"
+            - {{ join "," .Values.additionalNamespaces | quote }}
+            {{- end }}
+            {{- if $.Values.privateKeyAnnotations }}
+            {{- $privatekeyAnnotations := ""}}
+            {{- range $k, $v := $.Values.privateKeyAnnotations }}
```

Are we using here the right context? Both `.Values` calls are outside the `range`, unless I'm mistaken we should use the following:

```suggestion
            {{- if .Values.privateKeyAnnotations }}
            {{- $privatekeyAnnotations := ""}}
            {{- range $k, $v := .Values.privateKeyAnnotations }}
```

Same comment applies to the logic used for `privateKeyLabels` below.